src/main/java/io/stargate/sgv2/jsonapi/api/model/command/clause/filter/FilterClause.java

```diff
@@ -17,7 +15,6 @@
 import org.eclipse.microprofile.openapi.annotations.enums.SchemaType;
 import org.eclipse.microprofile.openapi.annotations.media.Schema;
 
-@JsonDeserialize(using = FilterClauseDeserializer.class)
```

Comment:
When we make this change, will the SchemaValidatable and validate* functions from this class move to the deserializer ? They were a not good idea from me.

Reply:
Eventually yes something like that (or maybe exactly that).

Given that it looks like there is no validation for `FilterClause`, and that empty methods are called only from `FilterResolver`, one possibility would be to remove use of `SchemaValitable` at this point, and add back when we realize what to validate.
But it would follow from schema-specific deserializer split I think.

Reply:
Correction: there IS actual validation for `FilterClause` so cannot just drop the call.

Reply:
Use of `SchemaValitable` removed from `FilterClause` handling: still remains for `SortClause` (to be removed).